src/crypto/crypto_keys.cc

```diff
@@ -543,7 +543,6 @@ Maybe<bool> GetAsymmetricKeyDetail(
     case EVP_PKEY_EC: return GetEcKeyDetail(env, key, target);
     case EVP_PKEY_DH: return GetDhKeyDetail(env, key, target);
   }
-  THROW_ERR_CRYPTO_INVALID_KEYTYPE(env);
   return Nothing<bool>();
```

Comment:
Returning `Nothing<>` means that there’s an exception pending, so this should either not be changed or changed all the way (to e.g. `Just(false)`)

Reply:
What do you prefer?

- Moving the key type enum to JS and only call keyDetails() for known okay types?
- Returning `Just(false)`
- throwing and catching that throw in JS

Reply:
All of these are okay for me

Reply:
Thank you for the review.